qiita_db/util.py

```diff
@@ -579,9 +578,7 @@ def insert_filepaths(filepaths, obj_id, table, filepath_table,
         Id of the object calling the functions. Disregarded if move_files
         is False
     table : str
-        Table that holds the file data.
-    filepath_table : str
-        Table that holds the filepath information
+        Table that holds the file data
```

Comment:
Minor, but you accidentally deleted a period at the end of this sentence. :neckbeard: